docs/conf.py

```diff
@@ -245,3 +245,22 @@ def linkcode_resolve(domain, info):
         }
     }
 }
+
+# TODO: Change Function to check for versions that are greater than equal to 0.7.1
+# by comparing version numbers.
+def check_if_v071dev0():
+    """Check to see if current version being built is v0.7.1.dev0"""
+    result = False
+
+    if fairlearn.__version__ == "0.7.1.dev0":
+        print("Detected 0.7.1.dev0 in fairlearn.__version__")
+        result = True
+
+    return result
```

Comment:
Before the PR gets merged, we will have to change this version check. I was thinking of using a `greater than comparison` here. I did look into this and realized we might have to use the `packaging` library as suggested in [Stackoverflow](https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python). However I am not sure if we want to introduce another dependency?

Reply:
since this is a doc build dependency, I don't mind adding it.

Reply:
Yes, I later realized that packaging is already used by `setuptools` so most people should already have it installed. But to be explicit, I have added it to the `requirements-dev.txt`.